includes/InputToEntity/MinimalItemInput.php

```diff
@@ -113,6 +130,45 @@ public function getItem( array $inputEntity, PropertyId $reconcileUrlProperty )
 		return [ $item, array_values( $otherItems ) ];
 	}
 
+	/**
+	 * @param string $propertyId
+	 * @return PropertyId|null
+	 * @throws PropertyLabelNotResolvedException
+	 */
+	public function getPropertyId( string $propertyId ): ?PropertyId {
+		if ( strlen( $propertyId ) > 1 && $propertyId[0] === 'P' ) {
+			try {
+				return new PropertyId( $propertyId );
+			} catch ( \InvalidArgumentException $exception ) {
+				$this->logger->warning( 'Tried parsing something weird' );
```

Comment:
I don’t think this warrants a warning – it’s totally reasonable for a property label to start with `P` (Wikidata has [328 such properties](https://query.wikidata.org/#SELECT%20%2a%20WHERE%20%7B%0A%20%20%3Fp%20a%20wikibase%3AProperty%3B%0A%20%20%20%20%20rdfs%3Alabel%20%3FpLabel.%0A%20%20FILTER%28LANG%28%3FpLabel%29%20%3D%20%22en%22%29%0A%20%20FILTER%28STRSTARTS%28%3FpLabel%2C%20%22P%22%29%29%0A%7D) just in English). I think we should try to parse any string as a property ID first, and if that fails, then fall back to interpreting the string as a label, with at most an `->info()` log message (or maybe even just `->debug()`).

Reply:
And after interpreting it as a label do we just use the label in the info or debug message?

Reply:
This has been addressed. Tho I am not sure if what I did is what you meant. Could you check again?

Reply:
Yup, that looks good now. Though I feel like we could even remove the logger call completely, and then stop injecting a logger and save ourselves a bunch of code. WDYT?

Reply:
:shrug: ok!